### DISCUSSION TITLE
bitcoind service does not start after a while

### DISCUSSION THREAD

**Discussion by rene78** [raspiblitz/raspiblitz#4685] [Q&A] [answered]
Any idea what to do if one gets this message:
```
Blockchain Progress : 740 restarts
Lightning Progress  : 100.00 % 2 peers
```
![image](https://github.com/user-attachments/assets/a858d569-2e88-4bf7-b9d0-f20a5d0e5467)

After a restart it works for around half an hour again before the bitcoind service fails again.

This is my debug file.
https://termbin.com/o075

If I try to shut it down with 'sudo shutdown' the following error message appears in the display:
![image](https://github.com/user-attachments/assets/7acf7983-5c3d-4dd2-8e6f-449daaf978a0)

I guess my Raspiblitz had too many cold restarts.

**Comment by rootzoll** [comment, marked as answer]
It looks like your SSD got corrupted - which could have happend on a power cut out. If you dont have any funds on the device I would recommend to re-setup with a fresh sd card image of RaspiBlitz and reformat the SSD (delete all data and dont reuse blockchain).

If you still have problems with SSD after this it could be a permantent hardware issue with the SSD or the controller board.
